docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -134,6 +134,8 @@ cinder.csi.openstack.org   2019-07-29T09:02:40Z
 
 > NOTE: If using certs(`ca-file`), make sure to update the manifests (controller and node plugin) to mount the location of certs as volume onto container as well.
 
+> NOTE: refer to [livenessprobe](https://github.com/kubernetes-csi/livenessprobe) for how to enable liveness probe for CSI related containers.
```

Comment:
@jichenjc Its already enabled by default in our manifests with this PR right. This doesnt seem to be right place , its applicable for both manifests and helm.
Could we move the doc as features may be https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/features.md , and add some details on why liveness probe and where to configure values (link to the manifests)

Reply:
yes, that make sense , actually I want to avoid `features.md` at first glimpse as 
the features are all about `feature` and livenessprobe is not a `feature` (CSI related)..anyway, I think I agree with you now to update there..